File: csharp/Rectangle.cs
/*
 * Created by SharpDevelop.
 * User: GameBase
 * Date: 25/02/2018
 * Time: 17:56
 * 
 * To change this template use Tools | Options | Coding | Edit Standard Headers.
 */
using System;

namespace Dpo_project
{
	/// <summary>
	/// Description of Rectangle.
	/// </summary>
	public class Rectangle : AFigure
	{
		private int width;
		private int len;
		private float x;
		private float y;
		private float z;
		

		public Rectangle()
		{
		}
		public override void getColor() {}
		public override void setColor() {}
		public override void set() {}
		
		public void setAttr(float x, float y, float z) {
			this.x = x;
			this.y = y;
			this.z = z;
		}
		
		public override float getX() {
			return x;
		}
		public override float getY() {			
			return y;
		}
		public override float getZ() {
			return z;
		}
		
		public void setlen(int len) {
			this.len = len;
		}
		
		public void setWidth(int width) {
			this.width = width;
		}
		
		public override void dessiner(){
			
		}
		
		public override void calculerSurface() {
			
		}
	}
}

File: java/src/Rectangle.java
public class Rectangle extends AFigure {
    private float l;
    private float H;
    private float L;

    public Rectangle(float l, float H, float L) {
        this.l = l;
        this.H = H;
        this.L = L;
    }

    @Override
    public void calculerSurface() {

    }

    public float getl() {
        return l;
    }

    public void setl(float l) {
        this.l = l;
    }

    public float getL() {
        return L;
    }

    public void setL(float L) {
        this.L = L;
    }

    public float getH() {
        return H;
    }

    public void setH(float H) {
        H = H;
    }

    @Override
    public void dessiner() {
        System.out.println(String.format("Dessine un Rectangle (l: %f, H: %f, L: %f)",
                this.getl(), this.getH(), this.getL()));
    }
}
